src/nnet3/nnet-example.cc

```diff
@@ -56,23 +56,25 @@ bool NnetIo::operator == (const NnetIo &other) const {
 }
 
 NnetIo::NnetIo(const std::string &name,
-               int32 t_begin, const MatrixBase<BaseFloat> &feats):
+               int32 t_begin, const MatrixBase<BaseFloat> &feats,
```

Comment:
Please rename these t_subsampling_factor variables to t_stride, here and in the header.